cmd/README.md

```diff
@@ -0,0 +1,521 @@
+(only for DynamoDB). By default, the schema sample size is 100,000.
+
+### Target profile (`-target-profile`)
+
+HarbourBridge accepts the following options for --target-profile,
+specified as "key1=value1,key2=value,..." pairs:
+
+`-dbname` Specifies the name of the Spanner database to create. This must be a
+new database. If dbname is not specified, HarbourBridge creates a new unique
+dbname.
+
+`-instance` Specifies the Spanner instance to use. The new database will be
+created in this instance. If not specified, the tool automatically determines an
+appropriate instance using gcloud.
+
+## Example Usage
+
+Details on HarbourBridge example usage can be found here: 
+- [PostgreSQL example usage](sources/postgres/README.md#example-postgresql-usage)
+- [MySQL example usage](sources/mysql/README.md#example-mysql-usage)
+- [DynamoDB example usage](sources/dynamodb/README.md#example-dynamodb-usage)
+
+
+## Schema Conversion
+
+Details on HarbourBridge schema conversion can be found here:
+- [PostgreSQL schema conversion](sources/postgres/README.md#schema-conversion)
+- [MySQL schema conversion](sources/mysql/README.md#schema-conversion)
+- [DynamoDB schema conversion](sources/dynamodb/README.md#schema-conversion)
+
+## Data Conversion
+
+HarbourBridge converts PostgreSQL/MySQL/DynamoDB data to Spanner data based on 
+the Spanner schema it constructs. Conversion for most data types is fairly 
+straightforward, but several types deserve discussion. Note that HarbourBridge 
+is not intended for databases larger than about 100GB. Details on HarbourBridge 
+data conversion can be found here:
+- [PostgreSQL data conversion](sources/postgres/README.md#data-conversion)
+- [MySQL data conversion](sources/mysql/README.md#data-conversion)
+- [DynamoDB data conversion](sources/dynamodb/README.md#data-conversion)
+
+## Troubleshooting Guide
+
+HarbourBridge is written using the Go module system, and so it must be
+installed in [module-aware mode](https://golang.org/cmd/go/#hdr-Module_support).
+This can be achived by setting the environment variable `GO111MODULE` to `on`
+before calling "go get" e.g.
+
+```sh
+GO111MODULE=on go get github.com/cloudspannerecosystem/harbourbridge
+```
+
+As an alternative to "go get", you can make a copy of the HarbourBridge
+codebase from the github repository and use "go run":
+
+```sh
+git clone https://github.com/cloudspannerecosystem/harbourbridge
+cd harbourbridge
+pg_dump mydb > mydb.pg_dump
+go run github.com/cloudspannerecosystem/harbourbridge schema -source=postgresql < mydb.pg_dump
+```
+
+This workflow also allows you to modify or customize the HarbourBridge codebase.
+
+The following steps can help diagnose common issues encountered while running
+HarbourBridge.
+
+### 1. Verify driver configuration
+
+First, check that the driver is correctly configured to connect to your
+database. Driver configuration varies depending on the driver used.
+
+#### 1.1 pg_dump
```

Comment:
considering we have a DB specific readme, does it make sense to add it here ? We should just try to link it to the DB specific readme

Reply:
I agree that we should clean this up, but would prefer if we do that work as a follow up since it's independent from this PR.